ppr-ui/src/components/mhrRegistration/HomeOwners/AddHomeOwnerOrganization.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <div>
+    <v-card flat class="py-6 px-8 rounded">
+      <v-row id="mhr-home-manufacturer-name">
+        <v-col cols="3">
+          <label class="generic-label">
+            Add an Organization
+          </label>
+        </v-col>
+        <v-col cols="9">
+          <label class="generic-label" for="org-legal-name">
+            Organization Legal Name
+          </label>
+          <v-row>
+            <v-col>
+              <p>
+                You can look-up a B.C. business by entering the name of the
+                business or the incorporation number (including Societies and
+                extra-provincial companies registered in B.C.). If the name of
+                the organization does not appear in the look-up, enter the full
+                legal name of the organization.
+              </p>
+              <p>
+                <v-expansion-panels>
+                  <v-expansion-panel>
+                    <v-expansion-panel-header
+                      class="pa-0 primary--text"
+                      :hide-actions="true"
+                    >
+                      <div>
+                        <v-icon color="primary" v-on="on"
+                          >mdi-information-outline
+                        </v-icon>
+                        Help with Proprietorships / Partnerships
+                      </div>
+                    </v-expansion-panel-header>
+                    <v-expansion-panel-content class="help-toggle-content">
+                      Registered owners of a manufactured home cannot be a sole
+                      proprietorship, partnership or limited partnership. The
+                      home must be registered in the name of the proprietor or
+                      partner (person or organization).
+                    </v-expansion-panel-content>
+                  </v-expansion-panel>
+                </v-expansion-panels>
+              </p>
```

Comment:
Cool! I haven't see the use of expansion panels for help content yet. 

Do you find this is better UX than a simple click event to `v-show` the text content?
Or are they the same in funtionality?

Reply:
Reworked to use a simpler `v-show`.